simulators/eth2/withdrawals/specs.go

```diff
@@ -49,6 +49,7 @@ var (
 		AltairForkEpoch:         common.Big0,
 		BellatrixForkEpoch:      common.Big0,
 		CapellaForkEpoch:        common.Big1,
+		GenesisDelaySlots:       big.NewInt(10), // To avoid skipped slots in the start
```

Comment:
I think 10 slots might be too much, would 5 slots accomplish the same outcome?

This would add 1 minute to every test case run for each combination unfortunately.

Reply:
That actually depends upon the system load and use case. For this one, I believe `5` will work as well.